python/packages/autogen-core/src/autogen_core/_agent_id.py

```diff
@@ -44,8 +51,18 @@ def from_str(cls, agent_id: str) -> Self:
 
     @property
     def type(self) -> str:
+        """
+        An identifier that associates an agent with a specific factory function.
+
+        Strings may only be composed of alphanumeric letters (a-z) and (0-9), or underscores (_).
```

Comment:
This is no longer accurate - see: https://github.com/microsoft/autogen/blob/main/docs/design/02%20-%20Topics.md#identifiers

Reply:
This is accurate for AgentId